static/search.js

```diff
@@ -320,6 +320,7 @@ var Search = Control.extend({
 		if (index && currentIndexVersion === indexVersion) {
 			searchEngine = lunr.Index.load(index);
 		}else{
+			var dummyContainer = document.createElement('div');
```

Comment:
@chasenlehara would it be acceptable to use [`striptags`](https://github.com/ericnorris/striptags)? I can't find a better solution without using the DOM.